.github/workflows/wheels.yml

```
@@ -67,16 +67,19 @@ jobs:
         # Nb. keep cibuildwheel version pin consistent with job below
         run: pipx install cibuildwheel==3.2.0
       - id: set-matrix
+        # sed substitutions for ARM in the matrix below are unused (because ARM is done in the Stable ABI)
+        # but are left for convenience if we ever want to change that.
         run: |
           MATRIX=$(
             {
-              cibuildwheel --print-build-identifiers --platform linux \
+              cibuildwheel --print-build-identifiers --platform linux --archs x86_64 \
               | jq -nRc '{"only": inputs, "os": "ubuntu-latest"}' \
               | sed -e '/aarch64/s|ubuntu-latest|ubuntu-24.04-arm|' \
               && cibuildwheel --print-build-identifiers --platform macos \
               | jq -nRc '{"only": inputs, "os": "macos-latest"}' \
-              && cibuildwheel --print-build-identifiers --platform windows \
-              | jq -nRc '{"only": inputs, "os": "windows-2022"}'
+              && cibuildwheel --print-build-identifiers --platform windows --archs AMD64 \
+              | jq -nRc '{"only": inputs, "os": "windows-2022"}' \
+              | sed -e '/arm64/s|windows-2022|windows-11-arm|'
```

If you ask for `AMD64` specifically, then you don't need to accommodate Windows-ARM.
```suggestion
              && cibuildwheel --print-build-identifiers --platform windows --archs AMD64 \
              | jq -nRc '{"only": inputs, "os": "windows-2022"}' \
```